app/views/content/teacher-training-advisers/_article.html.erb

```diff
@@ -0,0 +1,110 @@
+    <p>
+      Teacher training advisers give you free, one-to-one support, whether you’re just thinking about teaching or you’re ready to apply.
+    </p>
+
+    <p>
+      They have years of teaching experience and can answer all your questions about life in the classroom, as well as how you can get there.
+    </p>
+    <%= render Content::QuoteComponent.new(
+      text: "They would answer any question I had, small or big, in a relaxed environment.",
+      background: "grey"
+    ) %>
+    <h2 class="heading--box-blue">How does it work</h2>
+    <p>You’ll be matched with your own dedicated adviser. You’ll have the same adviser throughout your journey into teaching.</p>
+
+    <p>How often you talk to them is up to you – this could be every week for more than a year, or just a few times for some one-to-one support.</p>
+
+    <p>You can chat by phone, or by:</p>
+
+    <ul>
+      <li>email</li>
+      <li>text</li>
+      <li>WhatsApp</li>
+    </ul>
+    <h2 class="heading--box-blue">How can your adviser help you?</h2>
+    <h3>If teaching is just an idea</h3>
+    <p>
+      Your adviser can talk to you about what teaching is really like and help you understand if it’s right for you.
+    </p>
+
+    <p>
+      They can:
+    </p>
+
+    <ul>
+      <li>talk through your questions and any doubts ​</li>
+      <li>help you find school experience</li>
+      <li>explain the steps to becoming a teacher</li>
+    </ul>
+
+    <h3>If you're ready to apply</h3>
+
+    <p>
+      Your adviser can talk you through the application process step by step.
+    </p>
+
+    <p>
+      They can support you with:
+    </p>
+
+    <ul>
+      <li>writing your personal statement</li>
+      <li>understanding your different training options</li>
+      <li>preparing for interviews</li>
+    </ul>
+
+      <%= render Content::QuoteListComponent.new(
+        quotes: [{
+          heading: "Habiba, teacher with a hearing impairment",
+          text: "The application process went smoothly thanks to the help of my teacher training adviser, Ellie.",
+          background: "grey",
+          accreditation: "<a href=\"/blog/overcoming-challenges-to-become-a-teacher-hearing-impairment\">Read how Habiba successfully applied to train to teach</a>",
+        }]
+      ) %>
+
+    <h3>If you've applied and been unsuccessful</h3>
```

Comment:
@plisovin @philspray 

The apostrophe on this page is rendering differently to those in production - can we change it to match prod (which is our usual style)?

Review app
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/0d461303-eb92-4229-bdf0-28f672c70edc)

Production
![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/2abef67e-7645-4b98-9d70-fdaff7b4c596)